webpack.config.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */ 
+
```

Comment:
enable eslint and fix the error flagged by eslint.

Reply:
- [ ] You didn't enable eslint as suggested by the previous TSE